### PR TITLE
Unittesting

### PR DESCRIPTION
**Pull request by GustavHansen99**
Please have a brief look at the test scripts and comment if you think it is done wrongly or if we should have more tests - saw these as the bare minimum at the moment at least :)

### CODE REVIEW

**Comment by GustavHansen99** [comment]
We should at least remember to revisit the tests when we have the complete pipeline in place

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        matrix:
+            os: [ubuntu-latest, macos-latest]
+            python-version: ["3.8", "3.10"]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
```
Comment (nikarav) [inline .github/workflows/tests.yml:23]: Is there any difference is we use `version 3` or `version 1` instead of `version 2`?

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    - uses: iterative/setup-dvc@v1
+
+    - name: Get data
+      run: dvc pull
+
+    - name: Test with pytest
+      run: |
+        pip install pytest
+        pytest -v
```
Comment (nikarav) [inline .github/workflows/tests.yml:54]: Does this also run the tests ?

File: .github/workflows/tests.yml (hunk 1 above)
Comment (GustavHansen99) [reply .github/workflows/tests.yml:23]: I'll have to pass on that one..

File: .github/workflows/tests.yml (hunk 2 above)
Comment (GustavHansen99) [reply .github/workflows/tests.yml:54]: This is the code that executes the three test scripts defined tests/ - yes :)

**Review by diangeli — APPROVED** [review]
LGTM

File: .github/workflows/tests.yml (hunk 2 above)
Comment (nikarav) [reply .github/workflows/tests.yml:54]: ohh niceee! :)

**Comment by nikarav** [comment]
Good job mate! 👍

**Review by nikarav — APPROVED** [review]
